components/VersionSection.tsx

```diff
@@ -2,19 +2,39 @@
 
 import { CheckCircle2, Info, Loader2, XCircle } from 'lucide-react';
 import { use, useEffect, useState } from 'react';
+import Markdown from 'react-markdown';
 import { z } from 'zod';
+import Link from '~/components/Link';
+import { Alert, AlertDescription, AlertTitle } from '~/components/ui/Alert';
 import { env } from '~/env';
 import { type getInstallationId } from '~/queries/appSettings';
 import { ensureError } from '~/utils/ensureError';
 import SettingsSection from './layout/SettingsSection';
 import Paragraph from './ui/typography/Paragraph';
 
+type SemVerUpdateType = 'major' | 'minor' | 'patch';
+
 const GithubApiResponseSchema = z.object({
-  status: z.string(),
-  ahead_by: z.number(),
-  behind_by: z.number(),
+  tag_name: z.string().transform((tag) => tag.replace('v', '')), // remove the v to compare with APP_VERSION
```

Comment:
I think it would be better to do this parsing in a RegExp, which could also extract the elements of the version.

While we are at it, why not use zod to parse this into an _actual_ semver? So we give it a string, and we get out an object with major, minor, and patch properties?